plugins/@grouparoo/postgres/.eslintrc.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  env: {
+    es2021: true,
+    node: true,
+  },
```

Comment:
We still need to support node.js v12... does this work?

Reply:
I guess it's just for the linter... so it's OK

Reply:
Typescript should transpile everything down so it doesn't matter I'd think. Either way we could always change this so we don't use 2021 features.